db/migrate/20230902155110_add_devise_to_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[7.0]
```

Comment:
- [ ] Well resolved 👍🏽 
I was able to migrate successfully, sign up, confirm my account, and sign in successfully 🥇 
<img width="1289" alt="Screenshot 2023-09-02 at 18 05 30" src="https://github.com/ab-noori/Blog-App/assets/58692590/b8b31d44-2073-45cd-a15e-71aa364d775b">